### ISSUE TITLE
Both REST /add-smartphone-to-cart/ and /add-to-cart/ not working

### ISSUE THREAD

**Issue by Pengu1no** [awesto/cookiecutter-django-shop#9]
## What happened?
I've created Angular 7 frontend application. Each endpoint worked well except cart adding. Sending POST request does not adds products to cart. When I tried to add `serializer.save()` into core `shop.views.catalog` I got errors and I have no idea how to proceed
P.S. User is authenticated

## What should've happened instead?
Product should appear in the cart

## Steps to reproduce
* Add new product to database
* Authenticate (ex. JWT)
* Send POST to add to cart

OR

* Add new product to database
* Authenticate (in-box authentication)
* Open /<slug>/add-smartphone-to-cart/
* Send any value in post

Profit!

[//]: # (Any or all of the following:)
## OS: Ubuntu 18.04, local run
[//]: # (* Host system configuration: OS, Docker & friends' versions etc.)
[//]: # (* Project generation options)
[//]: # (* Logs)

**Comment by Pengu1no** [comment]
P.P.S. Library code is not needed to be changed to reproduce issue

**Comment by jrief** [comment]
Could you please tell me which values you are sending to the endpoint `.../add-to-cart/`.
Did you compare these values with those sent by the browser, if the project has been set up through cookiecutter?

**Comment by Pengu1no** [comment]
Yep, I've followed [readme](https://github.com/awesto/cookiecutter-django-shop/blob/master/README.md#run-django-shop-demo-locally)

**Comment by Pengu1no** [comment]
My screenshots
![Screenshot 1 (rest)](https://user-images.githubusercontent.com/26686814/59750315-4d803980-9287-11e9-9e3e-e5502e37c41a.png)
![Screenshot 2 (rest)](https://user-images.githubusercontent.com/26686814/59750356-5cff8280-9287-11e9-83ee-bdc752aa8703.png)
![Screenshot Cart](https://user-images.githubusercontent.com/26686814/59750405-7274ac80-9287-11e9-8907-0f3b214bf448.png)
And since my "qwerty" product is smartphone I cannot use /add-to-cart/ (screenshot below)
![Error](https://user-images.githubusercontent.com/26686814/59750574-af40a380-9287-11e9-8c5a-c1952d881cab.png)
